cmd/package.go

```diff
@@ -0,0 +1,340 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/otiai10/copy"
+	"github.com/spf13/cobra"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+var install bool
+
+func init() {
+	rootCmd.AddCommand(packageCmd)
+	packageSnapCmd.Flags().BoolVarP(&install, "install", "i", false, "Install the snap after packaging it")
+	packageCmd.AddCommand(packageSnapCmd)
+	packageDebCmd.Flags().BoolVarP(&install, "install", "i", false, "Install the deb after packaging it")
+	packageCmd.AddCommand(packageDebCmd)
```

Comment:
Have you considered introducing a `hover install` command (like `flutter install`) ?